### DISCUSSION TITLE
Running client on mac or linux

### DISCUSSION THREAD

**Discussion by jradzikowski** [fedorovvl/tso_client#284] [Q&A] [answered]
Hi there!

is there a way to run the client on linux or mac os?

**Comment by Mazimus332** [comment]
Hello, yes

Linux: https://hub.docker.com/r/vladimirsirris/tsoclient
MAC: https://github.com/fedorovvl/tso_client/wiki/%D0%9A%D0%BB%D0%B8%D0%B5%D0%BD%D1%82-%D0%B8-%D0%BF%D0%BE%D0%B4%D1%81%D0%B2%D0%B5%D1%82%D0%BA%D0%B0-%D0%B4%D0%BB%D1%8F-Mac

Or join Discord: https://discord.gg/wxKaeWz3JE
And read FAQ

**Reply by jradzikowski** [reply]
I followed the instruction on linux, with docker and what I got is a tso client running in a window but I don't see anything like mundane warfare. Is it because I forgot to replace something?

**Reply by fedorovvl** [reply, marked as answer]
linux version doesn't support extended features

**Comment by jmmfr** [comment]
Hi.
A little late, but why "does not support extended features " ?
Regards.

**Reply by fedorovvl** [reply]
short version - i cant make it work =)
